lib/matplotlib/cbook/__init__.py

```diff
@@ -1150,7 +1150,7 @@ def boxplot_stats(X, whis=1.5, bootstrap=None, labels=None,
     def _bootstrap_median(data, N=5000):
         # determine 95% confidence intervals of the median
         M = len(data)
-        percentiles = [2.5, 97.5]
+        percentiles = (2.5, 97.5)
```

Comment:
I don't think this matters wrt speed (does it?); also I prefer a list here as this matches "more" the return type of np.percentile below (a ndarray is closer to a list than to a tuple.)